client/my-sites/plan-features-comparison/style.scss

```diff
@@ -335,24 +335,6 @@ $plan-features-sidebar-width: 272px;
 .plans-features-main__group.is-scrollable {
 	position: relative;
 
-	.is-section-signup & {
-		width: 100vw;
-		margin-left: calc( 50% - 50vw );
-
-		@media ( min-width: 1600px ) {
-			max-width: 1600px;
-			margin-left: -280px;
-		}
-
-		@include breakpoint-deprecated( '<1040px' ) {
-			padding-top: 12px;
-		}
-	}
-
-	.signup__steps & .plan-features--signup {
-		max-width: 100%;
-	}
-
```

Comment:
I removed all this CSS because it's identical to what we have in `client/my-sites/plan-features/style.scss`, and AFAICT these two stylesheets are always loaded together (`client/my-sites/plans-features-main/index.jsx` imports the components associated with both stylesheets).

@Automattic/martech if someone could take a glance at this to double check, that would be very helpful!

Reply:
This has been extracted into #67325